hrabsence/hrabsence.php

```diff
@@ -235,6 +235,8 @@ function hrabsence_civicrm_install() {
   );
   civicrm_api3('message_template', 'create', $msg_params);
 
+  _hrabsence_add_scheduled_jobs();
```

Comment:
I think it would be a better Idea to move anything inside this hook to the upgrader install() method , as well as any function you defined here could be moved to the upgrader class .. it is better than polluting civicrm with more global functions and make this file easier to understand ... I already did the same in same data extension and thinking to to do the same for anything I work on .